projects/vdk-core/src/vdk/internal/builtin_plugins/run/cli_run.py

```diff
@@ -141,6 +141,24 @@ def __log_exec_result(self, execution_result: ExecutionResult) -> None:
         else:
             log.info(f"Data Job execution summary: {execution_result}")
 
+        self.__log_short_job_summary(execution_result)
```

Comment:
This code will log two job summaries if we set LOG_EXECUTION_RESULT=True https://github.com/vmware/versatile-data-kit/blob/dd916d25adce6ce0596016f2d1cb34023936746f/projects/vdk-core/src/vdk/internal/builtin_plugins/run/cli_run.py#L183C1-L183C1

The call to `__log_short_job_summary` should be in this else clause instead.
https://github.com/vmware/versatile-data-kit/blob/dd916d25adce6ce0596016f2d1cb34023936746f/projects/vdk-core/src/vdk/internal/builtin_plugins/run/cli_run.py#L185

Reply:
The new summary will only be shown when the old one is shown, do we want it to appear instead of the old one?
The else clause here is irrelevant to this, the if-else here is a bugfix for jobs with massive step lists having their summary not print on some OS-es.

Reply:
addressed comment